domain/src/main/java/com/example/domain/Address.java

```diff
@@ -0,0 +1,24 @@
+package com.example.domain;
+
+import jakarta.persistence.*;
```

Comment:
In every model we need javax.persistence not jakarta
Also try to avoid wildcard imports (import only what you use)